src/styles/blocks/card.scss

```
@@ -0,0 +1,100 @@
+.card {
+  display: flex;
+  flex-direction: column;
+
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
```

create color for all variables
```suggestion
  border: 1px solid $c-someColor;
```